examples/run-blockchain/test-data.json

```diff
@@ -0,0 +1,159 @@
+{
+  "blocks": [
+    {
+      "blockHeader": {
+        "bloom": "00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000",
+        "coinbase": "8888f1f195afa192cfee860698584c030f4c9db1",
+        "difficulty": "0x020000",
+        "extraData": "0x",
+        "gasLimit": "0x023ec6",
+        "gasUsed": "0x021536",
+        "hash": "f53f268d23a71e85c7d6d83a9504298712b84c1a2ba220441c86eeda0bf0b6e3",
+        "mixHash": "29f07836e4e59229b3a065913afc27702642c683bba689910b2b2fd45db310d3",
+        "nonce": "8957e6d004a31802",
+        "number": "0x01",
```

Comment:
The other option is to stop supporting anything before homestead. I really think this is the best option. We would need to `grep -r homestead` in ethereumjs-vm and ethereumjs-block and remove all homestead conditionasl. (there shouldn't be to many)

@axic if we start doing this do you think it would be a good time to back port tx, and block type change from ewasm?